purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -117,6 +118,10 @@ fun Purchases.getOfferingsWith(
  * @param [onSuccess] Will be called after the purchase has completed
  * @param [onError] Will be called after the purchase has completed with error
  */
+@Deprecated(
+    "Replaced with purchaseProductWithOption",
+    ReplaceWith("purchaseProductWithOption")
+)
```

Comment:
Still not sure about whether we should deprecate this or remove it. But as before, will leave these for now.